packages/autorest.python/test/vanilla/legacy/specification/internaloperation/README.md

```diff
@@ -15,7 +15,7 @@ output-artifact: code-model-v4-no-tags
 payload-flattening-threshold: 1
 black: true
 vanilla: true
-package-mode: dataplane
+package-mode: azure-dataplane
```

Comment:
the code is kind of confusing, but we can't break `package-mode` for generations from swagger, but we can break it for generations from `tsp`. So in swagger, it should still be `package-mode: dataplane`, but in tsp-config.yaml, users should now have to input `package-mode: azure-dataplane`

Reply:
OK. Let us keep backward compatibility so that https://github.com/Azure/azure-sdk-for-python/pull/32169 is not needed